smf/sled-agent/non-gimlet/config.toml

```diff
@@ -88,3 +88,9 @@ level = "info"
 mode = "file"
 path = "/dev/stdout"
 if_exists = "append"
+
+# These are pre-generated keys and roots designed to be used for testing.
+# Create your own if you need to test
```

Comment:
```suggestion
# Create your own if you need to test a multi-node system.
```

I think that in almost all cases where we use sprockets we'll have multiple nodes, such as in a4x2. In these cases it will be a little weird for each node to have the same keys. Should we generate a bunch of extras and check them in, or provide notes about how to do so?

Reply:
I'll copy over the sled2 keys used in sprockets as well